DominicGBauer/system-design-dashboard
# feat: add highlights

## Scope
https://dev.azure.com/BRXDOM001/Dream%20Team%20Dashboardin/_sprints/taskboard/Dream%20Team%20Dashboardin%20Team/Dream%20Team%20Dashboardin/Sprint%204?workitem=94

## Work Done
1. Added in static highlights bar component to top.
2. Filled in some dummy data. 
3. Added testing for static version of this (note: will need to reword this when it becomes dynamic)

## Steps to Test

1. Pull branch 94_add-highlights
2. Run ```npm run start```
3. Check that highlights bar is there.

## Screenshot

![image](https://user-images.githubusercontent.com/80746660/119274440-24ede080-bc10-11eb-9f63-a563151ef425.png)

![image](https://user-images.githubusercontent.com/80746660/119274443-2f0fdf00-bc10-11eb-9ec1-eb12fceb0a06.png)
